core/src/main/java/org/apache/lucene/queries/SearchAfterSortedDocQuery.java

```diff
@@ -51,13 +49,13 @@
     private final int[] reverseMuls;
 
     public SearchAfterSortedDocQuery(Sort sort, FieldDoc after) {
-        if (sort.getSort().length != after.fields.length) {
+        if (sort.getSort().length < after.fields.length) {
             throw new IllegalArgumentException("after doc  has " + after.fields.length + " value(s) but sort has "
```

Comment:
There is an extra space between `doc` and `has`. Can you remove it while you are here?